src/NuGet.LibraryModel/LibraryDependencyType.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.LibraryModel
+{
+    public class LibraryDependencyType
+    {
+        private readonly LibraryDependencyTypeFlag[] _keywords;
+
+        public static LibraryDependencyType Default;
+
+        static LibraryDependencyType()
+        {
+            Default = new LibraryDependencyType(LibraryDependencyTypeKeyword.Default.FlagsToAdd as LibraryDependencyTypeFlag[]);
+        }
+
+        public LibraryDependencyType()
+        {
+            _keywords = new LibraryDependencyTypeFlag[0];
+        }
+
+        private LibraryDependencyType(LibraryDependencyTypeFlag[] flags)
```

Comment:
Even when they are constructors? I think that organizing members by type (Constructor/Property/Method/etc) trumps organizing by visiblity. I think this should be right by the other (public) constructor.